packages/ord-connect/src/hooks/useSignMessage.tsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from "react";
+import { useOrdContext } from "../providers/OrdContext.tsx";
+import getSignedMessage from "../lib/signMessage.ts";
+
+export function useSignMessage(): [
```

Comment:
And here I actually would prefer to remove forced return type and go with inferred version.

Reply:
updated !